TYPO3.Fluid/Classes/TYPO3/Fluid/ViewHelpers/Format/CurrencyViewHelper.php

```diff
@@ -75,13 +91,16 @@ class CurrencyViewHelper extends AbstractLocaleAwareViewHelper
      * @param string $currencySign (optional) The currency sign, eg $ or €.
      * @param string $decimalSeparator (optional) The separator for the decimal point.
      * @param string $thousandsSeparator (optional) The thousands separator.
+     * @param boolean $prependCurrency (optional) Indicates if currency symbol should be placed before or after the numeric value.
+     * @param boolean $separateCurrency (optional) Indicates if a space character should be placed between the number and the currency sign.
```

Comment:
`$currencySeparator` would probably be more in line with the other params for separation characters